.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "assets/Font-Awesome"]
+	path = assets/Font-Awesome
+	url = https://github.com/FortAwesome/Font-Awesome.git
+[submodule "assets/bootstrap-sass"]
+	path = assets/bootstrap-sass
+	url = https://github.com/twbs/bootstrap-sass
```

Comment:
Yeah, see the comment above. It would work if you could just include the cache files in git.